src/utils/common/common_test.c

```diff
@@ -198,7 +198,7 @@ DEF_TEST(escape_slashes) {
   for (size_t i = 0; i < STATIC_ARRAY_SIZE(cases); i++) {
     char buffer[32];
 
-    strncpy(buffer, cases[i].str, sizeof(buffer));
+    strcpy(buffer, cases[i].str);
```

Comment:
What if the source exceeds the buffer?

Reply:
It doesn't, since the source is defined in the test case itself.
I agree, it's not ideal in any case. I'm open to better suggestions here.